blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/widget-list.jsx

```diff
@@ -7,16 +7,10 @@ import {
 } from "../nav-helper";
 import NavWidget from "./nav-widget";
 
-const WidgetList = ({
-	breakpoint,
-	children,
-	customFields,
-	id,
-	menuButtonClickAction,
-	placement,
-}) => {
+const WidgetList = ({ children, customFields, id, menuButtonClickAction, placement }) => {
 	// istanbul ignore next
-	if (!id || !breakpoint) return null;
+	const breakpoint = "desktop";
```

Comment:
In the future, remove the breakpoint hard-coded here. I kept this string in there as a placeholder to make sure that two "widgets" weren't rendered.